pkg/discovery/probe/pod_probe_util.go

```diff
@@ -56,6 +56,27 @@ func GetResourceRequest(pod *api.Pod) (cpuRequest float64, memRequest float64, e
 	return
 }
 
+// Get the UUID that will be used in Turbonomic server. Here we build the UUID based on pod UID, namespace and name.
+// The out UUID is in format "UID:namespace:name"
+func GetTurboPodUUID(pod *api.Pod) string {
```

Comment:
It is a checking for safety.  Currently it depends on Kubernetes' implementation that the character ":" is illegal for the name.   Anyway,  this checking is only necessary if kubernetes may relax this restrictions, which is not very possible.